specification/sql/resource-manager/Microsoft.Sql/2014-04-01/examples/QueriesList.json

```diff
@@ -0,0 +1,68 @@
+{
+  "parameters":{
+    "subscriptionId":"00000000-1111-2222-3333-444444444444",
+    "resourceGroupName":"group1",
+    "serverName":"sqlcrudtest",
+    "databaseName": "db1",
+    "api-version":"2014-04-01"
+  },
+  "responses":{
+    "200":{
+      "value":[
+        {
+          "type":"Microsoft.Sql/servers/databases/topQueries",
+          "numberOfTopQueries":5,
```

Comment:
Example is not correct. If numberOfTopQueries is 5 "queries" list below must have 5 items and it has only one.